scss/tests/mixins/_utilities-map.test.scss

```diff
@@ -0,0 +1,339 @@
+        utilities-get-options("flex"),
+        (
+          responsive: true,
+          property: flex,
+          values: (fill: 1 1 auto)
+        ));
+    }
+
+    @include it("Return null if the utility does not exist") {
+      @include before ();
+      @include assert-equal(
+        utilities-get-options("non-existing"),
+        null
+      );
+    }
+  }
+
+  @include describe("utilities-get-option") {
+    @include it("Returns the option for the given utility") {
+      @include before ();
+      @include assert-equal(
+        utilities-get-option(flex, responsive),
+        true
+      );
+    }
+
+    @include it("Returns null if the option is not present") {
+      @include before ();
+      @include assert-equal(
+        utilities-get-option(flex, non-existing),
+        null
+      );
+    }
+
+    @include it("Returns null if the utility is not present") {
+      @include before ();
+      @include assert-equal(
+        utilities-get-option(non-existing, responsive),
+        null
+      );
+    }
+  }
+
+  @include describe("utilities-get-value") {
+    @include describe("utilities with map of values") {
+      @include it("Returns the value for the given utility") {
+        @include before ();
+        @include assert-equal(
+          utilities-get-value(flex, fill),
+          1 1 auto
+        );
+      }
+
+      @include it("Returns null if the value does not exist for the utility") {
+        @include before ();
+        @include assert-equal(
+          utilities-get-value(flex, non-existing),
+          null
+        );
+      }
+    }
+
+    @include describe("utilities with list of values") {
+      @include it("Returns the value if it is in the list") {
+        @include before ();
+        @include assert-equal(
+          utilities-get-value(align, top),
+          top
+        );
+      }
+
+      @include it("Returns null if the value is not in the list") {
+        @include before ();
+        @include assert-equal(
+          utilities-get-value(align, non-existing),
+          null
+        );
+      }
+    }
+
+    @include describe("utilities with single value") {
+      @include it("Returns the value if it is the one set") {
+        @include before ();
+        @include assert-equal(
+          utilities-get-value(margin, auto),
+          auto
+        );
+      }
+
+      @include it("Returns null if it is not the value set") {
+        @include before ();
+        @include assert-equal(
+          utilities-get-value(margin, non-existing),
+          null
+        );
+      }
+    }
+  }
+
+  @include describe("utilities-set-options") {
+    @include it("Merges the options with the existing ones") {
+      @include before ();
+
+      @include utilities-set-options(flex, (
+        responsive: false,
+        rtl: true
+      ));
+
+      @include assert-equal(utilities-get-option(flex, responsive), false, "responsive");
+      @include assert-equal(utilities-get-option(flex, rtl), true, "rtl");
```

Comment:
I'd have checked that the initial values are still here, for instance, `@include assert-equal(utilities-get-option(flex, property), flex, "property");` + Not sure that using another function is a good practice for unit test but I let you decide here !

Reply:
Good shout on checking that the function doesn't just wipe out the configuration 🙌🏻 

Regarding not re-using a function, it's not really textbook unit testing to re-use a function to assert, that's true. The function reused is quite light, tested right before so I think it's pretty safe here, especially as it makes the test more readable.

Reply:
Yes OK, I'm fines with that!